amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -68,7 +68,7 @@ export function* filterWorker(): SagaIterator {
   const state = yield select(getSearchState);
   const { search_term, selectedTab, filters } = state;
   const pageIndex = getPageIndex(state)
-  yield put(searchResource(search_term, selectedTab, pageIndex));
+  yield put(searchResource("filter", search_term, selectedTab, pageIndex));
```

Comment:
TODO: Will make these enums in the next upload, and rethink better names. i.e. "external_page_view" is more accurately when the search occurs from pulling parameters from the url -- which could also happen on refresh or manually updating url -- so will consider a different name.

Reply:
Can we use the redux action here? Most workers have only one redux action, but in this case it would have two: `UpdateSearchFilter.CLEAR_CATEGORY` and `UpdateSearchFilter.UPDATE_CATEGORY`. Not sure how to access the value in the worker.

Reply:
Not sure I understand. `searchResource()` is an action.